tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -150,6 +150,12 @@ COMMON_FLAGS := \
   $(CC_WARNINGS) \
   $(ADDITIONAL_DEFINES)
 
+ifneq ($(TARGET), xtensa)
+  ifneq ($(TARGET), hexagon)
```

Comment:
let's make this:
```make
# no-stringop-overread is not a valid flag for some toolchains
ifeq ($(findstring $(TARGET),xtensa hexagon),)
  COMMON_FLAGS += -Wno-stringop-overread
endif
```

that will make it easier to add more targets, if needed.

Reply:
Great suggestion! Learned it. Thank you!